KsApi/models/User.swift

```diff
@@ -121,6 +122,7 @@ extension User: Decodable {
     self.location = try? values.decodeIfPresent(Location.self, forKey: .location)
     self.name = try values.decode(String.self, forKey: .name)
     self.needsFreshFacebookToken = try values.decodeIfPresent(Bool.self, forKey: .needsFreshFacebookToken)
+    self.needsPassword = try values.decodeIfPresent(Bool.self, forKey: .needsPassword)
```

Comment:
Also this isn't just a property to decode, it can also be encoded like `facebookConnected`, check `extension User: EncodableType {` in this file. The case where the User object is encoded is `updateUserSelf` in `Service`.